environment.yml

```diff
@@ -12,6 +12,7 @@ dependencies:
   - pip:
     - git+https://github.com/MAAP-Project/stac_ipyleaflet.git@0.3.6
     - jupyter-sshd-proxy
+    - jupytergis==0.3.0
```

Comment:
should we bump to latest and get it from [conda-forge](https://anaconda.org/conda-forge/jupytergis)? 0.6.2